epochStart/metachain/baseRewards.go

```diff
@@ -494,8 +494,8 @@ func getMiniBlockWithReceiverShardID(shardId uint32, miniBlocks block.MiniBlockS
 	return nil
 }
 
-func createBroadcastTopic(shardC sharding.Coordinator, destShId uint32) string {
-	transactionTopic := factory.RewardsTransactionTopic + shardC.CommunicationIdentifier(destShId)
+func createBroadcastTopic(topic string, shardC sharding.Coordinator, destShId uint32) string {
+	transactionTopic := topic + shardC.CommunicationIdentifier(destShId)
```

Comment:
Yep, reverted, probably I did it because it is a base...go file and I prepared this method to be used also by validatorInfo with its topic but this is used only for rewards.